### DISCUSSION TITLE
Apply Avro Schema Without Schema Registry Connection

### DISCUSSION THREAD

**Discussion by miguellgramacho96** [provectus/kafka-ui#1305] [Q&A] [answered]
Hi!

I'm currently storing the avro schema in the aws glue schema registry.
How can I (if its even possible) apply a avro schema to all of the messages from one topic, if I dont connect the kafka-ui to aws glue schema registry? (I think kafka-ui does not support aws glue).

Kind regards.

**Comment by miguellgramacho96** [comment, marked as answer]
Thaks @Haarolean .
Do you have an example on how to connect a aws glue shema registry with kafka-ui?

**Reply by Haarolean** [reply]
Hey, sorry for the confusing, it was the fake news 😅
Actually we have to implement [something](https://github.com/awslabs/aws-glue-schema-registry) to make this happen.
I'll convert this to an issue.

**Reply by miguellgramacho96** [reply]
Thanks @Haarolean .
The same goes for kafka connect, aws has their manage service - msk connect - but for that i don't believe there's a library, maybe just cli operations.

**Reply by Haarolean** [reply]
@miguellgramacho96 we do fully support MSK, but, regarding MSK connect -- not quite sure, will [take a look](https://github.com/provectus/kafka-ui/issues/1311).

**Comment by WhiskyTangoFawks** [comment]
We're running into the same issue, trying to visualise data with the schema from glue.

**Reply by Haarolean** [reply]
We'll planning on implementing Glue support in the next release

**Reply by mbuotidem** [reply]
@Haarolean Any news on Glue and MSK Connect support?

**Reply by cdia8890** [reply]
Checking about the possibility adding AWS Glue support. Thank you!
